rega-genotype/src/main/java/rega/genotype/BlastAnalysis.java

```diff
@@ -535,17 +536,22 @@ private Result compute(SequenceAlignment analysisDb, AbstractSequence sequence)
 
                     if (start == Integer.MAX_VALUE)
                     	start = -1;
-
+                    
+                    
+                    
```

Comment:
Please remove all the changes to the whitespaces that have no effect on this file